content/app/development/logic/validation/_index.en.md

```diff
@@ -365,19 +363,19 @@ private void ValidateBoAdresse(flyttemelding model, ModelStateDictionary validat
     {
         validationResults.AddModelError(
             nameof(model.boaddresse), 
-            "Boadresse kan ikke være lengere enn 150 tegn.");
+            "Address can not be longer than 150 characters.");
     }
 }
 ```
 
-### Spesifisere at valideringsfeil er fikset
-Når validering trigges av et enkelt felt, så vil alle tidligere valideringer på dette feltet fjernes i påvente av svar fra den siste valideringen.
-Dersom et felt trigger validering som oppdaterer/legger til feilmelding på flere felter på en gang, vil ikke disse fjernes selv om det ikke lenger er noen
-feil i disse feltene. Dette er fordi man ikke har noen måte å vite hvilke felter som ev. er validert ifm en enkeltfeltvalidering.
+### Specify that validation errors are fixed
+When validation is triggered by a single field, all former validations on this field will be removed pending a response from the last validation.
+If a field triggers validation that updates/adds an error message to multiple fields at once, these will not be removed even though there is no longer any
+error in these fields. This is because there is no way to know which fields may have been validated through a single field validation.
```

Comment:
```suggestion
If a field triggers validation that updates/adds an error message to multiple fields at once, these will not be removed even when there no longer are any
errors in these fields. This is because there is no way to know which fields may have been validated through a single field validation.
```